<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-FS5UPGRD" ignored="0" name="masterTest" tests="7" failures="5" timestamp="2023-03-20T19:58:57 IST" time="108.678" errors="0">
  <testcase name="test_acc_registration" time="1.185" classname="testcases.TC_001AccountRegistration"/>
  <testcase name="loginvalid" time="26.264" classname="testcases.Tc_002login"/>
  <testcase name="test_login" time="0.725" classname="testcases.TC_003ddtLogintest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testcases.TC_003ddtLogintest.test_login(TC_003ddtLogintest.java:58)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
  <testcase name="test_login" time="0.016" classname="testcases.TC_003ddtLogintest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testcases.TC_003ddtLogintest.test_login(TC_003ddtLogintest.java:58)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
  <testcase name="test_login" time="0.036" classname="testcases.TC_003ddtLogintest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testcases.TC_003ddtLogintest.test_login(TC_003ddtLogintest.java:58)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
  <testcase name="test_login" time="0.02" classname="testcases.TC_003ddtLogintest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testcases.TC_003ddtLogintest.test_login(TC_003ddtLogintest.java:58)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
  <testcase name="test_login" time="0.017" classname="testcases.TC_003ddtLogintest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testcases.TC_003ddtLogintest.test_login(TC_003ddtLogintest.java:58)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test_login -->
</testsuite> <!-- masterTest -->
